docs/developerguide.rst

```diff
@@ -56,3 +56,135 @@ We are using Prettier to format our code. In order to use it before a Pull Reque
    .. code-block:: console
 
       $ prettier --write **/*.js
+
+CERN
+----
+
+Prerequisites
+~~~~~~~~~~~~~
+
+You must run the Minikube cluster with VirtualBox VM so you listen requests from outside the
+VM. In order to configure the port forwading, go to the VirtualBox UI and select
+the running VM, then in the *Network* section select *Advanced* > *Port Forwarding*
+
+.. image:: _static/virtualbox-port-forwarding.png
+
+Add a new entry (in the screenshot ``reana-backend``) to forward all the requests going to the port
+443 (default for HTTPS) to the internal port 30443 (where REANA-Server is listening).
+
+Authentication
+~~~~~~~~~~~~~~
+REANA-UI uses Single Sign-On (SSO) for Authentication.
+
+1. `Register <https://sso-management.web.cern.ch/OAuth/RegisterOAuthClient.aspx>`_ an OAuth client.
+
+  * ``client_id``: A unique text string that identifies your client, e.g. ``my-reana-ui``.
+  * ``redirect_id``: Your REANA-Server hostname followed by ``/oauth/authorized/cern/``,
+    e.g. ``https://myreana.cern.ch/oauth/authorized/cern/``
+  * Application Homepage: Your REANA-Server hostname, e.g. ``https://myreana.cern.ch``.
+  * Description: Whatever description you like.
+
+2. Edit REANA-UI's `config.js <https://github.com/reanahub/reana-ui/blob/master/reana-ui/src/config.js#L14>`_:
+  * ``api``: REANA-Server hostname, e.g. ``https://myreana.cern.ch``.
+
+3. Configure SSO-related REANA-Cluster environment variables. You can do this directly on the
+   `cluster configuration template <https://github.com/reanahub/reana-cluster/blob/master/reana_cluster/configurations/reana-cluster-minikube-dev.yaml>`_
```

Comment:
we normally call this file by its generic name so [`reana-cluster.yaml`](https://github.com/reanahub/reana-cluster/blob/master/reana_cluster/configurations/reana-cluster-minikube-dev.yaml) instead of `cluster configuration template`.